### PR TITLE
fix: remove outlook protection for miro deep link

### PR DESCRIPTION
**Pull request by splagemann**
TLDR
----
This will remove the automatically ingested outlook protection link when deep linking into Miro.

More Detail
---
MS is automatically changing links in MS Teams or Outlook and prefixes them with a safety protection check. This basically destroys deep links into any application that is not a browser. I know that simply removing the link part is not a very clean solution, but by using the `miroapp` protocol handler and checking for miro.com as part of the url I think it should be ok from a security perspective.

There is maybe a more general solution for other deeplinks and/or other safety protection urls, so consider this as a first idea to address this problem.

This would solve one of my daily use-cases, but I do understand if this is considered a workaround and shouldn't go into this application.

Regards

### CODE REVIEW

**Comment by will-stone** [comment]
Hi @splagemann 👋🙂 

Hmmm, I'm not sure. Have you tried [Finicky](https://github.com/johnste/finicky)?

I have Finicky set up to handle something similar on my work machine, but for Teams links:

```js
module.exports = {
  defaultBrowser: "Browserosaurus",
  handlers: [
    /**
     * Microsoft Teams
     */
    {
      match: finicky.matchHostnames(["teams.microsoft.com"]),
      browser: "com.microsoft.teams",
      url: ({ url }) => ({ ...url, protocol: "msteams" }),
    },
  ],
};
```

**Comment by splagemann** [comment]
Hey, thanks for the answer. Interesting, so you're basically chaining those tools. Will experiment with that! Thanks

**Comment by will-stone** [comment]
Yep! I basically set Finicky as my default browser, and then B as Finicky's default 🤓 That way I get the rules engine _and_ the UI selector.

**Comment by splagemann** [comment]
Works for me! Closed this PR
